<h1 align="center">
<b>Lecture 04</b><br>
De novo genome assembly
</h1>
<p align="center">
<b>Date:</b> Sep 5, 2024
</p>

!!! danger "DRAFT"

    This page is a work in progress and is subject to change at any moment.

This lecture explores the intricate world of de novo genome assembly, a powerful approach for reconstructing genomes without prior reference. We'll unravel the complex puzzle of piecing together complete genomes from short DNA fragments, highlighting the unique challenges this method presents. The session introduces key concepts like contigs and scaffolds, essential building blocks in the assembly process. We'll examine fundamental assembly algorithms, with a focus on the overlap-layout-consensus approach, and discuss the obstacles posed by genomic complexities such as repetitive sequences. By interpreting common quality metrics, students will gain practical skills in assessing assembly accuracy and completeness. This comprehensive overview aims to equip students with a deep understanding of de novo assembly techniques and their critical role in expanding our genomic knowledge.

## Learning objectives

What you should be able to do after today's lecture:

1.  Explain the concept of de novo genome assembly and how it differs from reference-guided assembly.
2.  Describe the fundamental challenge of reconstructing a complete genome from short, overlapping reads.
3.  Define and differentiate between contigs and scaffolds in the context of genome assembly.
4.  Outline the basic steps of a de novo assembly algorithm, focusing on the overlap-layout-consensus approach.
5.  Discuss key challenges in de novo assembly, such as repetitive sequences and heterozygosity.
6.  Interpret basic assembly quality metrics, such as N50 and total assembly length, using a simple example.

## Readings

Relevant content for today's lecture.

TODO:

## Presentation

TODO:
