<h1 align="center">
<b>Lecture 09</b><br>
Gene expression quantification
</h1>
<p align="center">
<b>Date:</b> Sep 24, 2024
</p>

We'll explore how raw sequencing reads are transformed into meaningful measures of gene activity, navigating the complexities of multi-mapped reads and isoform variations.
The session will compare various quantification metrics, from traditional RPKM to more recent innovations like TPM, highlighting their strengths and limitations.
We'll examine cutting-edge tools for transcript-level quantification and discuss the crucial role of normalization in generating comparable expression data across samples.
Through practical examples, students will learn to interpret gene expression results, bridging the gap between computational output and biological insight.

## Learning objectives

What you should be able to do after today's lecture:

1.  Discuss the importance of normalization and quantification in RNA-seq data analysis.
2.  Explain the relevance of pseudoalignment instead of read mapping.
3.  Understand the purpose of Salmon's generative model.
4.  Describe how salmon handles experimental biases in transcriptomics data.
5.  Communicate the principles of inference in Salmon.

## Readings

Relevant content for today's lecture.

-   [Problem formulation](https://omics.crumblearn.org/transcriptomics/quant/problem/)
-   [Salmon](https://omics.crumblearn.org/transcriptomics/quant/salmon/#maximum-likelihood)

## Presentation

**Download:** [biosc1540-l09.pdf](/lectures/09/biosc1540-l09.pdf)
