<h1 align="center">
<b>Lecture 06</b><br>
Multiple sequence alignment
</h1>
<p align="center">
<b>Date:</b> Sep 12, 2024
</p>

!!! danger "DRAFT"

    This page is a work in progress and is subject to change at any moment.

This lecture explores the powerful technique of multiple sequence alignment (MSA), a vital tool for unraveling evolutionary relationships and functional similarities among genes and proteins. We'll journey beyond pairwise comparisons to tackle the complexities of aligning numerous sequences simultaneously. The session illuminates how MSA algorithms, such as the widely-used ClustalW and MUSCLE, navigate the intricate landscape of multiple sequence relationships. By examining guide trees and progressive alignment methods, students will gain insight into the strategies employed to manage the computational challenges inherent in MSA. Through hands-on analysis of alignment outputs, we'll learn to identify conserved regions and infer potential evolutionary connections, showcasing MSA's crucial role in modern genomics and molecular biology research.

## Learning objectives

What you should be able to do after today's lecture:

1.  Define multiple sequence alignment (MSA) and explain its importance in comparative genomics and evolutionary studies.
2.  Differentiate between pairwise and multiple sequence alignments, highlighting the increased complexity and challenges of MSA.
3.  Describe the progressive alignment method, including the concept of guide trees and how they are used in MSA algorithms.
4.  Explain the basic principles of popular MSA algorithms such as ClustalW or MUSCLE.
5.  Discuss the challenges in MSA, including computational complexity and handling of insertions/deletions across multiple sequences.
6.  Interpret and analyze a simple multiple sequence alignment output, identifying conserved regions and potential evolutionary relationships.

## Readings

Relevant content for today's lecture.

TODO:

## Presentation

TODO:
